utils/fetchPoolUserInfo.ts

```diff
@@ -0,0 +1,39 @@
+import { CENNZAsset } from "@/types";
+import { Api } from "@cennznet/api";
+import { Balance } from "@/utils";
+import { PoolUserInfo } from "@/types";
+
+/**
+ * Fetch user share of a liquidity pool
+ *
+ * @param {string} api
+ * @param {string} userAddress
+ * @param {CENNZAsset} tradeAsset
+ * @param {CENNZAsset} coreAsset
+ * @return {Promise<PoolUserInfo>}
+ */
+export default async function fetchPoolUserInfo(
+	api: Api,
+	userAddress: string,
+	tradeAsset: CENNZAsset,
+	coreAsset: CENNZAsset
+): Promise<PoolUserInfo> {
+	const [{ asset, core }, currentLiquidity] = await Promise.all([
+		await (api.rpc as any).cennzx.liquidityValue(
```

Comment:
do we need any? is api (v 2.*) able to get the type definition for rpc calls?

Reply:
gonna loop back and check with you on this, as with a few other `rpc` methods, we have this error when not adding `any` in

![Screen Shot 2022-03-31 at 09 07 02](https://user-images.githubusercontent.com/97480229/160921845-e3e04c25-016a-4dd8-8c69-2c7669c75fa1.png)